--- package/cmake/cmake.mk	2017-09-01 16:17:43.000000000 -0700
+++ package/cmake/cmake.mk	2018-01-12 18:17:45.691980210 -0800
@@ -9,6 +9,7 @@
 CMAKE_SITE = https://cmake.org/files/v$(CMAKE_VERSION_MAJOR)
 CMAKE_LICENSE = BSD-3-Clause
 CMAKE_LICENSE_FILES = Copyright.txt
+CMAKE_MAKE = $(MAKE1)
 
 # CMake is a particular package:
 # * CMake can be built using the generic infrastructure or the cmake one.
