--- package/Config.in	2017-09-01 16:17:43.000000000 -0700
+++ package/Config.in	2018-01-11 23:02:47.007938491 -0800
@@ -27,6 +27,7 @@
 	source "package/jack2/Config.in"
 	source "package/kodi/Config.in"
 	source "package/lame/Config.in"
+	source "package/libspotify/Config.in"
 	source "package/madplay/Config.in"
 	source "package/mimic/Config.in"
 	source "package/miraclecast/Config.in"
@@ -322,6 +323,7 @@
 	source "package/gtkperf/Config.in"
 	source "package/leafpad/Config.in"
 	source "package/midori/Config.in"
+	source "package/navit/Config.in"
 	source "package/nodm/Config.in"
 	source "package/pcmanfm/Config.in"
 	source "package/qt-webkit-kiosk/Config.in"
@@ -478,6 +480,7 @@
 	source "package/rtl8821au/Config.in"
 	source "package/sane-backends/Config.in"
 	source "package/sdparm/Config.in"
+	source "package/ssd1306/Config.in"
 	source "package/setserial/Config.in"
 	source "package/sg3_utils/Config.in"
 	source "package/sigrok-cli/Config.in"
