--- package/rpi-userland/rpi-userland.hash	2018-01-12 10:35:14.504363961 -0800
+++ package/rpi-userland/rpi-userland.hash	2017-09-01 16:17:43.000000000 -0700
@@ -1,2 +1,2 @@
 # Locally computed
-sha256 46a37eaf0de598ce5804e48a5efc2120e07d753a57ed83463ebfa551562eefe6 rpi-userland-29835184aa70c13d77a11d721e934a7298bdf46a.tar.gz
+sha256 476cde13d0d4bbf44ffc4d1d545847464d5c270bb72e4d940e43e5972c4de9da rpi-userland-29835184aa70c13d77a11d721e934a7298bdf46a.tar.gz
